charts/ingress-nginx/templates/controller-daemonset.yaml

```diff
@@ -110,6 +110,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+          {{- if .Values.controller.enableMimalloc }}
+            - name: LD_PRELOAD
```

Comment:
Shouldn't this also be added to the deployment?

Reply:
Yes. Fixed.